components/Popover/Popover.stories.tsx

```diff
@@ -62,3 +65,39 @@ export const RichContent: ComponentStory<typeof PopoverForStory> = (args) => (
     </PopoverForStory>
   </Container>
 );
+
+export const IconTrigger: ComponentStory<typeof PopoverForStory> = (args) => (
+  <Container>
+    <PopoverForStory {...args}>
+      <PopoverTrigger asChild>
+        <Button size="small">
+          <HamburgerMenuIcon />
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent arrowCss={{ color: '$primary' }} css={{ bc: '$primary', p: '$2' }}>
+        <Text css={{ c: 'white' }}>Content</Text>
```

Comment:
I think we could maybe use another color/variable here to make it more legible in the dark theme, `$loContrast` here should work 👌 

![Screen Shot 2022-03-14 at 12 06 59](https://user-images.githubusercontent.com/38889364/158201336-ac8f3689-64d4-4c60-b700-9ce3e9b9cb27.png)